hadoop-cloud/src/hadoop-3/main/scala/org/apache/spark/internal/io/cloud/PathOutputCommitProtocol.scala

```diff
@@ -161,7 +204,16 @@ object PathOutputCommitProtocol {
   val REJECT_FILE_OUTPUT_DEFVAL = false
 
   /** Error string for tests. */
-  private[cloud] val UNSUPPORTED: String = "PathOutputCommitProtocol does not support" +
+  private[cloud] val UNSUPPORTED: String = "PathOutputCommitter does not support" +
     " dynamicPartitionOverwrite"
 
+  /**
+     * Stream Capabilities probe for spark dynamic partitioning compatibility.
```

Comment:
indentation?

Reply:
fixed. interesting. i've switched my IDE to rendering scala/javadocs and it hid that. something for me to worry about. maybe I should turn it off again